extensions_admin/pulp_rpm/extensions/admin/repo_options.py

```diff
@@ -51,6 +58,14 @@ def parse_skip_types(t):
 d = d % {'t': ', '.join(VALID_SKIP_TYPES)}
 OPT_SKIP = PulpCliOption('--skip', d, required=False, parse_func=parse_skip_types)
 
+d = _('Allow unsigned packages. Defaults to True')
+OPT_ALLOW_UNSIGNED = PulpCliOption('--allow-unsigned', d, required=False,
+                                   parse_func=parsers.parse_boolean)
+
+d = _('Allow signature keys. Comma separated keys.')
+OPT_ALLOW_KEYS = PulpCliOption('--allow-keys', d, required=False, allow_multiple=False,
```

Comment:
Would this be better as an `allow_multiple=True` option? I'm not sure which is better.